discovery-provider/src/tasks/index_aggregate_user.py

```diff
@@ -360,16 +370,32 @@ def update_aggregate_table(
                 f"index_aggregate_user.py | most_recent_indexed_aggregate_block: {most_recent_indexed_aggregate_block}"
             )
 
+            elapsed_time = get_elapsed_time_redis(redis,
```

Comment:
instead of adding the notion of time here, might be easier to keep everything consistent with blocks and just do latest block - last refreshed block > TWO_WEEKS_IN_BLOCKS but don't feel super strongly about this

Reply:
Yeah this is way better